TYPO3.Flow/Classes/TYPO3/Flow/Core/ClassLoader.php

```diff
@@ -234,33 +245,19 @@ protected function loadClassFromPossiblePaths(array $possiblePaths, array $names
     /**
      * Sets the available packages
      *
-     * @param array $allPackages An array of \TYPO3\Flow\Package\Package objects
      * @param array $activePackages An array of \TYPO3\Flow\Package\Package objects
      * @return void
      */
-    public function setPackages(array $allPackages, array $activePackages)
+    public function setPackages(array $activePackages)
     {
         /** @var Package $package */
-        foreach ($allPackages as $packageKey => $package) {
-            if (isset($activePackages[$packageKey])) {
-                if ($package->getAutoloadType() === Package::AUTOLOADER_TYPE_PSR4) {
-                    $this->createNamespaceMapEntry($package->getNamespace(), $package->getClassesPath(), self::MAPPING_TYPE_PSR4);
-                } else {
-                    $this->createNamespaceMapEntry($package->getNamespace(), $package->getClassesPath());
-                }
-                if ($this->considerTestsNamespace) {
-                    $this->createNamespaceMapEntry($package->getNamespace(), $package->getPackagePath(), self::MAPPING_TYPE_PSR4);
-                }
-            } else {
-                // Remove entries coming from composer for inactive packages.
-                if ($package->getAutoloadType() === Package::AUTOLOADER_TYPE_PSR4) {
-                    $this->removeNamespaceMapEntry($package->getNamespace(), $package->getClassesPath(), self::MAPPING_TYPE_PSR4);
-                } else {
-                    $this->removeNamespaceMapEntry($package->getNamespace(), $package->getClassesPath());
-                }
-                if ($this->considerTestsNamespace) {
-                    $this->removeNamespaceMapEntry($package->getNamespace(), $package->getPackagePath(), self::MAPPING_TYPE_PSR4);
-                }
+        foreach ($activePackages as $packageKey => $package) {
```

Comment:
I'd only do `$activePackages as $package`, you do not use `$packageKey` AFAIK :)